assets/scss/case-studies.scss

```diff
@@ -1,8 +1,16 @@
 // Case Studies & Interactive Storytelling Styles - Premium Modern Design
+@import "util/colors";
+
+/* Hero color variant for solutions/news */
+.hero.solutions {
```

Comment:
This should be set via the Hugo page parameters, see other pages. And if the color is missing, add it in the correct location.